tempto-examples/src/main/resources/sql-tests/datasets/sample_jdbc_table.ddl

```diff
@@ -1,5 +1,5 @@
 -- type: jdbc
 CREATE TABLE %NAME% (
-  id INT,
+  id BIGINT,
```

Comment:
That's why I said it's unrelated change. ;) It could be a separate commit. Though, it's such a small change it's not worth it.